app/models/manageiq/providers/container_manager.rb

```diff
@@ -17,6 +17,7 @@ class ContainerManager < BaseManager
     has_many :container_component_statuses, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_builds, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_build_pods, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :container_templates, :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
In regards to the last commen (https://github.com/ManageIQ/manageiq/pull/11217/files#r78644713), add
```
has_many :container_templates, :dependent => :destroy , through: :container_project
```